lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/grid/query/CellValueFilter.json

```diff
@@ -0,0 +1,78 @@
+{
+	"description": "A filter used to select rows based on cell values. For example, to handle a user request like 'find all rows where the Project column is Alpha', you would set 'columnName' to 'Project', 'operator' to 'EQUALS', and 'value' to ['Alpha']",
+	"type": "object",
+	"implements": [
+		{
+			"$ref": "org.sagebionetworks.repo.model.grid.query.Filter"
+		}
+	],
+	"properties": {
+		"columnName": {
+			"type": "string",
+			"description": "Required. The name of the column to filter by."
+		},
+		"operator": {
+			"type": "string",
+			"description": "The comparison operator.",
+			"name": "CellValueOperator",
+			"id": "org.sagebionetworks.repo.model.grid.query.CellValueOperator",
+			"enum": [
+				{
+					"name": "EQUALS",
+					"description": ""
+				},
+				{
```

Comment:
This would work in JSON Schema to add a description/title, but probably not in schema-to-pojo

```
"oneOf": [
  {"const": "EQUALS", "title": "..."},
  {"const": "NOT_EQUALS", "title": "..."}
]
```